### DISCUSSION TITLE
ImportError: cannot import name 'DIFFUSERS_SLOW_IMPORT' from 'diffusers.utils'

### DISCUSSION THREAD

**Discussion by Neel0101** [huggingface/diffusers#7515] [Q&A] [answered]
In the newest version, when using: from diffusers import AutoencoderKL, DDPMScheduler, UNet2DConditionModel , getting the error : ImportError: cannot import name 'DIFFUSERS_SLOW_IMPORT' from 'diffusers.utils' (/opt/conda/lib/python3.10/site-packages/diffusers/utils/__init__.py)

System Info : 
diffusers 0.27.2

Need help.

**Comment by tolgacangoz** [comment]
Hi @Neel0101,
I think there is a problem with your environment. Could you clear your cache and then reinstall:
```sh
python -m pip uninstall diffusers && conda uninstall diffusers
conda clean -a
python -m pip cache purge
python -m pip install diffusers
```

**Reply by Neel0101** [reply]
doesn't help, i still get the same error.

**Reply by tolgacangoz** [reply]
Are you in a jupyter-notebook?

**Reply by Neel0101** [reply]
yes

**Reply by tolgacangoz** [reply, marked as answer]
After running these:
```sh
!python -m pip uninstall diffusers -y && conda uninstall diffusers -y
!conda clean -ay
!python -m pip cache purge
```
Restart the jupyter-notebook kernel, then run this:
```python
!python -m pip install diffusers
from diffusers import AutoencoderKL, DDPMScheduler, UNet2DConditionModel
```
Let's see what will happen. If this won't work, I suggest you create a new conda environment and try on it. I think you are using the base environment of conda, maybe other libraries you installed before conflict somehow 🤔. Also, could you share the output of this command:
```sh
diffusers-cli env
```

**Reply by Neel0101** [reply]
you were right, restarting kernel after running above commands helped, thanks a bunch!

**Reply by tolgacangoz** [reply]
I am glad you found the answer 🤗.

**Comment by MohamedLahmeri01** [comment]
@tolgacangoz , i am facing the same issue, i tried clone a huggingface repo , but got issue , where the diffusion is not installed , whatever i do nothing seems to be working : here is the link : https://colab.research.google.com/drive/1roi0OaxdDDNbRlwtcHx-eyi2JntbrjhS?usp=sharing
